server/src/test/java/org/candlepin/resource/ConsumerResourceUpdateTest.java

```diff
@@ -770,7 +770,9 @@ public void canUpdateConsumerEnvironment() {
     @Test(expected = NotFoundException.class)
     public void throwsAnExceptionWhenEnvironmentNotFound() {
         String uuid = "A Consumer";
-        EnvironmentDTO changedEnvironment = new EnvironmentDTO("42", "environment", null);
+        EnvironmentDTO changedEnvironment = new EnvironmentDTO();
+        changedEnvironment.setId("42");
+        changedEnvironment.setName("environment");
 
```

Comment:
isn't the place to chain the attriubte setting?
changedEnvironment.setId("42").setName("environment");

Reply:
Done.